spec/models/saved_job_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe SavedJob, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
@NicoleKMarino we're missing this test. We can test here the relationships between SavedJob and User and Job